core/src/main/java/com/predic8/membrane/core/interceptor/oauth2/processors/EmptyEndpointProcessor.java

```diff
@@ -40,7 +46,21 @@ public boolean isResponsible(Exchange exc) {
     }
 
     @Override
-    public Outcome process(Exchange exc) throws Exception {
+    public Outcome process(Exchange exc) {
+        try {
+            return processInternal(exc);
+        } catch (Exception e) {
+            log.error(e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.